test/Makefile

```diff
@@ -31,6 +31,7 @@ SUCCESS_TESTS = \
 	for-nested.py \
 	global.py \
 	if.py \
+	int.py \
```

Comment:
Looks like you may have forgotten to include 'int.py'?

Reply:
I haven't added because it needs python > 3.5 and I wasn't sure if we could bump this requirement. Thanks for catching it. Will add.

Reply:
I think it's fine to require a newer python to test against; if we need to allow that, we can add checks for the python version in the build system.